docs/source/conf.py

```diff
@@ -25,7 +25,7 @@
 release = "21.2"  # telegram.__version__
 
 # If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = "6.1.3"
```

Comment:
why is this required?

Reply:
just updated it because it seemed quite outdated

Reply:
If we don't have a fixed need for that, I don't see a strict reason to do that 🤔  in any case, I would rather have that on doc-fixes …